.travis.yml

```diff
@@ -32,12 +32,14 @@ env:
     - DEP_VERSION=v0.5.4
 
 before_install:
+  - mkdir -p $GOPATH/bin
   - curl
     https://raw.githubusercontent.com/golang/dep/${DEP_VERSION}/install.sh|sh
 
 jobs:
   include:
     - name: static-check-make
+      arch: amd64
```

Comment:
why this is required?

Reply:
This is not required actually, as amd64 is the default arch. I'm putting it here just to make clear arm64 build doesn't run these tests. They can be removed.